modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+The amount of resources required by Network Observability depends on the size of your cluster and your requirements for ingesting and storing observability data. There are settings to consider that can help you get an optimal setup and observability.
+
+The following settings can help you manage resources and performance from the outset:
```

Comment:
I think that this lead in sentence could fuse with the sentence before it. I see the word "help" used twice.

Reply:
Maybe "help you" might be a slight form of [anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism). 

How about:

> To manage resources and set performance criteria for your cluster, consider configuring the following settings. Configuring these settings might meet your optimal setup and observability needs.

Reply:
Addressed